src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -0,0 +1,196 @@
+window.addEventListener('portalactivate', evt => {
+  // Data available as evt.data
+  const data = evt.data;   
+});
+```
+## Retrieving the predecessor
+```javascript
+// Listen to the portalactivate event
+window.addEventListener('portalactivate', evt => {
+  // ... and creatively use the predecessor
+  const portal = evt.adoptPredecessor();
+  document.querySelector('someElm').appendChild(portal);
+});
+```
+## Knowing your page was adopted as a predecessor
+```javascript
+// The activate function returns a Promise.
+// When the promise resolves, it means that the portal has been activated.
+// If this document was adopted by it, then window.portalHost will exist.
+portal.activate().then(_ => {
+  // Check if this document was adopted into a portal element.
+  if (window.portalHost) {
+    // You can start communicating with the portal element
+    // i.e. listen to messages
+    window.portalHost.addEventListener('message', evt => {
+      // handle the event
+    });
+  }
+});
+```
+By combining all the features supported by Portals, you can build really fancy user experiences. For instance the demo below demonstrates how Portals can enable a seamless user experience between a website and a third party embed content. 
+
+<div style="width:100%; padding-top: 56.25%; position: relative;">
+  <iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://www.youtube.com/embed/4JkipxFVE9k" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+[Here is the link to the GitHub repo](https://github.com/WICG/portals/tree/master/demos/portal-embed-demo) and you can play around with it. Be creative and have fun!!
+
+# Use cases and the plan
+I hope you liked this brief tour of Portals! We can’t wait to see what you can come up with. For instance, you might want to start using Portals for non-trivial navigations such as: prerendering the page for your best-seller product from a product category listing page. Another important thing to know is that Portals can be used in cross-origin navigations, just like iframes. So, if you have multiple websites that cross reference one another, you can also use Portals to create seamless navigations between two different websites. This cross-origin use case is very unique to Portals, and can even improve the user experience of SPA.
+
+Note that Portals is still in early stages so not everything is working yet (that’s why it’s behind an experimental flag). That said, it’s ready for early experimentation in Chrome Canary. Feedback from the community is crucial to the design of new APIs, so please try it out and tell us what you think! You can check the current limitations [here](https://bugs.chromium.org/p/chromium/issues/detail?id=957836) and if you have any feature requests, or feedback, please head over to the [WICG GitHub repo](https://github.com/WICG/portals/issues).
```

Comment:
```suggestion
Note that Portals are still in early stages so not everything is working yet (that’s why it’s behind an experimental flag). That said, it’s ready for early experimentation in Chrome Canary. Feedback from the community is crucial to the design of new APIs, so please try it out and tell us what you think! You can check the current limitations on [the Chromium bug tracker](https://bugs.chromium.org/p/chromium/issues/detail?id=957836) and if you have any feature requests, or feedback, please head over to the [WICG GitHub repo](https://github.com/WICG/portals/issues).
```

Reply:
Suggest not using `here` as link text for accessibility reasons.
https://web.dev/labels-and-text-alternatives/#links

Reply:
Understood! I haven't really thought of a11y so thanks for pointing out!!